app/build.gradle

```diff
@@ -1,29 +1,43 @@
 apply plugin: 'com.android.application'
+
 apply from: '../checkci.gradle'
 
 android {
-    compileSdkVersion 26
-    defaultConfig {
-        applicationId "com.duycuong.weather"
-        minSdkVersion 15
-        targetSdkVersion 26
-        versionCode 1
-        versionName "1.0"
-        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
-    }
-    buildTypes {
-        release {
-            minifyEnabled false
-            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
-        }
+  compileSdkVersion 26
+  defaultConfig {
+    applicationId "com.duycuong.weather"
+    minSdkVersion 15
+    targetSdkVersion 26
+    versionCode 1
+    versionName "1.0"
+    testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+    buildConfigField "String", "DASK_SKY_API_KEY", "\"${DASK_SKY_API_KEY}\""
+    buildConfigField "String", "GEOCODE_API_KEY", "\"${GEOCODE_API_KEY}\""
+  }
+  buildTypes {
+    release {
+      minifyEnabled false
+      proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
     }
+  }
+  dataBinding {
+    enabled = true
+  }
 }
 
 dependencies {
-    implementation fileTree(dir: 'libs', include: ['*.jar'])
-    implementation 'com.android.support:appcompat-v7:26.1.0'
-    implementation 'com.android.support.constraint:constraint-layout:1.0.2'
-    testImplementation 'junit:junit:4.12'
-    androidTestImplementation 'com.android.support.test:runner:1.0.1'
-    androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.1'
+  implementation fileTree(dir: 'libs', include: ['*.jar'])
+  implementation 'com.android.support:appcompat-v7:26.1.0'
+  implementation 'com.android.support.constraint:constraint-layout:1.0.2'
+  implementation 'com.android.support:support-v4:26.1.0'
+  compile 'com.google.code.gson:gson:2.8.1'
```

Comment:
change compile to implementation
https://stackoverflow.com/questions/44493378/whats-the-difference-between-implementation-and-compile-in-gradle